plugins/system/skeletonkey/media/js/backend.js

```diff
@@ -76,7 +76,7 @@ const initSkeletonKey = () => {
 						return;
 					}
 
-					window.location = paths.rootFull;
+					window.open(paths.rootFull, '_blank');
```

Comment:
I am going to make this an option instead of having this take place by default. The reason is that most browsers block popups by default.

Moreover, I am wondering whether this change makes sense. Once you click that button your session is replaced with a new one which logs you in as the new user. You can no longer use the admin page unless you log back into the site (in which case your frontend login as a different user ceases to work). What is the purpose of this change?

Reply:
NIcholas,

I didn't even notice the rounded edges (but now do that you pointed it out). The icon I was using is already in use with core Joomla (in the upper right of the admin bar at the top) so figured it would be more future-proof than an older fa - class. 

I added the open in a new tab feature because I don't have shared sessions enabled so whether it was the intention or not, my admin session is still active in the original tab while I am logged in as the user in the front end.  In most of my use cases, I am logging in to the front end as a customer to see what they see (usually for a UI or customer service issue) and it makes it easier for me to toggle between tabs to adjust any backend settings without reopening another tab all over again.

If you don't believe anybody else would benefit from this then disregard and I appreciate the opportunity nonetheless. I have been a long-time subscriber to Akeeba and other extensions and I appreciate all that you do for the community.